Project 1/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>   
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel= "stylesheet" href= "styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="card"> 
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+<h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so intelligent.
+By Carl Zimmer</h2><h3>Nov. 30, 2018</h3>
+<figure>
+    <img src="https://static01.nyt.com/images/2018/12/04/science/04SCI-ZIMMER/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-articleLarge.jpg?quality=75&auto=webp&disable=upscale" alt="OctopusOnRocks">
+</figure><div class="card">
+   <p>To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus pulling two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like nesting bowls — and carts them away.
+
+<p>“It suggests the octopus is carrying these tools around because it has some understanding they may be useful in the future,” said Mr. Amodio, a graduate student studying animal intelligence at the University of Cambridge in Britain.
+
+<p>But his amazement is mixed with puzzlement.
+
+<p>For decades, researchers have studied how certain animals evolved to be intelligent, among them apes, elephants, dolphins and even some birds, such as crows and parrots.
+
+<p>But all the scientific theories fail when it comes to cephalopods, a group that includes octopuses, squid and cuttlefish. Despite feats of creativity, they lack some hallmarks of intelligence seen in other species.
+
+<pullquote><p>“It’s an apparent paradox that’s been largely overlooked in the past,”</pullquote> said Mr. Amodio. He and five other experts on animal intelligence explore this paradox in a paper published this month in the journal Trends in Ecology and Evolution.
```

Comment:
As discussed on Tuesday, you want:
1. for your pull quote to be a repetition of a quote in the text and not inline in the flow of the reading
2. you want to use this markup for your pull quote:
```
<aside>
  <blockquote>
    Your quote goes here.
  <blockquote>
</aside>
```